TransferMethod/Sources/UpdateTransferMethodPresenter.swift

```diff
@@ -0,0 +1,313 @@
+                        view.notifyTransferMethodUpdated(transferMethod)
+                    }
+                })
+            }
+        }
+    }
+
+    func prepareSectionForScrolling(_ section: UpdateTransferMethodSectionData,
+                                    _ row: Int,
+                                    _ focusWidget: AbstractWidget) {
+        section.rowShouldBeScrolledTo = row
+        section.fieldToBeFocused = focusWidget
+    }
+
+    private func errorHandler(for error: HyperwalletErrorType) {
+        switch error.group {
+        case .business:
+            resetErrorMessagesForAllSections()
+            if let errors = error.getHyperwalletErrors()?.errorList, errors.isNotEmpty {
+                updateFooterContent(errors)
+                if errors.contains(where: { $0.fieldName == nil }) {
+                    view?.showError(error,
+                                    pageName: UpdateTransferMethodPresenter.updateTransferMethodPageName,
+                                    pageGroup: UpdateTransferMethodPresenter.updateTransferMethodPageGroup,
+                                    nil)
+                }
+            }
+
+        default:
+            view?.showError(error,
+                            pageName: UpdateTransferMethodPresenter.updateTransferMethodPageName,
+                            pageGroup: UpdateTransferMethodPresenter.updateTransferMethodPageGroup) { [weak self] in
+                                self?.updateTransferMethod()
+            }
+        }
+    }
+
+    private func buildHyperwalletTransferMethod(_ transferMethod: HyperwalletTransferMethod)
+        -> HyperwalletTransferMethod? {
+        let transferMethodTypeCode = transferMethod.type ?? ""
+        let token = transferMethod.token ?? ""
+        switch transferMethodTypeCode {
+        case HyperwalletTransferMethod.TransferMethodType.bankAccount.rawValue,
+             HyperwalletTransferMethod.TransferMethodType.wireAccount.rawValue :
+            let bankAccount = HyperwalletBankAccount.Builder(token: token)
+                .build()
+            return bankAccount
+
+        case HyperwalletTransferMethod.TransferMethodType.bankCard.rawValue :
+            let bankCard = HyperwalletBankCard.Builder(token: token)
+                .build()
+            return bankCard
+
+        case HyperwalletTransferMethod.TransferMethodType.payPalAccount.rawValue:
+            let payPal = HyperwalletPayPalAccount.Builder(token: token)
+                .build()
+            return payPal
+
+        case HyperwalletTransferMethod.TransferMethodType.venmoAccount.rawValue:
+            let venmo = HyperwalletVenmoAccount.Builder(token: token)
+                .build()
+            return venmo
+
+        case HyperwalletTransferMethod.TransferMethodType.paperCheck.rawValue:
+            let paperCheck = HyperwalletPaperCheck.Builder(token: token)
+                .build()
+            return paperCheck
+
```

Comment:
update this to use transferMethodToken and HyperwalletTransferMethodConfiguration

Reply:
Updated